jest.config.packages.ts

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
All of these settings should match `jest.config.js`, aside from `collectCoverageFrom` and `moduleNameMapper` (explained below).